internal/aez/ccd_power_regulators.go

```diff
@@ -101,3 +103,30 @@ func (cpru *CPRU) Report() CPRUReport {
 		VOD3:   cpru.VOD3.voltage(),
 	}
 }
+
+//CSVSpecifications returns the specs used in creating the struct
+func (cpru CPRU) CSVSpecifications() []string {
+	return []string{"AEZ", Specification}
+}
+
+//CSVHeaders returns the field names
+func (cpru CPRU) CSVHeaders() []string {
+	val := reflect.Indirect(reflect.ValueOf(cpru.Report()))
```

Comment:
is is impossible to make a general CSVheaders function that you call from inside this and all similar funtions? so you do not need to "duplicate" code.

Reply:
For just the Headers it might be quite possible, for the values it's less so... I'll check if I can figure something out.